app/features/workout/WorkoutActions.js

```diff
@@ -23,6 +23,6 @@ export const restoreRep = (setID, repIndex) => ({
     removed: false
 });
 
-export const endSet = () => {
-    return SetsActionCreators.endSet();
+export const endSet = (manuallyStarted = true) => {
+    return SetsActionCreators.endSet(true, 0, manuallyStarted);
```

Comment:
Why are you passing true for sanity check here?

Reply:
Removed